docs/getting-started.md

```diff
@@ -36,7 +36,7 @@ gpg --recv-keys "<fingerprint>"
 ### Installing via Git
 
 ``` bash
-$ git clone git://github.com/bcoin-org/bcoin.git
+$ git clone https://github.com/bcoin-org/bcoin
```

Comment:
Can you update the link on README.md in root directory as well? I think that's the only other place.

Reply:
Sure. On it

Reply:
Done from my side.

Reply:
Do you have any more suggestions on the change @pinheadmz ?